docs/DeveloperGuide.md

```diff
@@ -268,16 +268,45 @@ _{Explain here how the data archiving feature will be implemented}_
 
 ### User stories
 
-Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unlikely to have) - `*`
-
-| Priority | As a …​                                    | I want to …​                     | So that I can…​                                                        |
-| -------- | ------------------------------------------ | ------------------------------ | ---------------------------------------------------------------------- |
-| `* * *`  | new user                                   | see usage instructions         | refer to instructions when I forget how to use the App                 |
-| `* * *`  | user                                       | add a new person               |                                                                        |
-| `* * *`  | user                                       | delete a person                | remove entries that I no longer need                                   |
-| `* * *`  | user                                       | find a person by name          | locate details of persons without having to go through the entire list |
-| `* *`    | user                                       | hide private contact details   | minimize chance of someone else seeing them by accident                |
-| `*`      | user with many persons in the address book | sort persons by name           | locate a person easily                                                 |
+|Status      | Stakeholder                   |Action                                                                                         |Effect                                                                                       |
```

Comment:
i think we should follow their headers and make the necessary changes to our user stories.

Reply:
| **Priority** | As a …​                                    | I want to …​                     | So that I can...

Actually can ba, I can change but please help me confirm what header should be used for  first column, ### **priority or status**?

Do we  need to modify the issues in github project if we changes userguide here, will be q much ah HAHA? or just devguide.